trunk/SUAVE/Components/Energy/Converters/Propeller.py

```diff
@@ -86,611 +82,338 @@ def spin(self,conditions):
-          body.transform_to_inertial (rotation matrix)
-          inertial.velocity_vector   [m/s]
-        conditions.propulsion.
-          throttle                   [-]
-          pitch_command              [radian/s] 
-
-        Outputs:
-        conditions.propulsion.acoustic_outputs.
-          number_sections            [-]
-          r0                         [m]
-          airfoil_chord              [m]
-          blades_number              [-]
-          propeller_diameter         [m]
-          drag_coefficient           [-]
-          lift_coefficient           [-]
-          omega                      [radian/s]
-          velocity                   [m/s]
-          thrust                     [N]
-          power                      [W]
-          mid_chord_aligment         [m] (distance from the mid chord to the line axis out of the center of the blade)
-        conditions.propulsion.etap   [-]
-        thrust                       [N]
-        torque                       [Nm]
-        power                        [W]
-        Cp                           [-] (coefficient of power)
-
-        Properties Used:
-        self. 
-          number_blades              [-]
-          tip_radius                 [m]
-          hub_radius                 [m]
-          twist_distribution         [radians]
-          chord_distribution         [m]
-          mid_chord_aligment         [m] (distance from the mid chord to the line axis out of the center of the blade)
-          thrust_angle               [radians]
+        self.inputs.omega                    [radian/s]
+        conditions.freestream.               
+          density                            [kg/m^3]
+          dynamic_viscosity                  [kg/(m-s)]
+          speed_of_sound                     [m/s]
+          temperature                        [K]
+        conditions.frames.                   
+          body.transform_to_inertial         (rotation matrix)
+          inertial.velocity_vector           [m/s]
+        conditions.propulsion.               
+          throttle                           [-]
+                                             
+        Outputs:                             
+        conditions.propulsion.outputs.        
+           number_radial_stations            [-]
+           number_azimuthal_stations         [-] 
+           disc_radial_distribution          [m]
+           thrust_angle                      [rad]
+           speed_of_sound                    [m/s]
+           density                           [kg/m-3]
+           velocity                          [m/s]
+           disc_tangential_induced_velocity  [m/s]
+           disc_axial_induced_velocity       [m/s]
+           disc_tangential_velocity          [m/s]
+           disc_axial_velocity               [m/s]
+           drag_coefficient                  [-]
+           lift_coefficient                  [-]
+           omega                             [rad/s]
```

Comment:
I think we should keep it as omega solely because it is used in so many textbooks. Also most of the references we get the equations from use omega. We can talk about this more over the next meeting if you would like